packages/transformers/svgo/src/SVGOTransformer.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import {Transformer} from '@parcel/plugin';
+
+import SVGO from 'svgo';
+
+const defaultConfig = {
+  plugins: [{prefixIds: true}],
+};
+
+export default new Transformer({
+  async getConfig({asset}) {
+    let svgoConfig = await asset.getConfig(
+      [
+        '.svgorc',
+        '.svgorc.json',
+        '.svgorc.yaml',
+        '.svgorc.yml',
+        'svgo.config.js',
+        '.svgo.yml',
+      ],
+      {
+        packageKey: 'svgo',
+      },
+    );
+
+    svgoConfig = svgoConfig || {};
+    svgoConfig = {...defaultConfig, ...svgoConfig};
+
+    this.svgo = new SVGO(svgoConfig);
```

Comment:
Not entirely sure if this is a good idea as it's not guaranteed that this state will be kept in other functions. It currently works but we could shift around a bit in transformrunner and break this.

Probably best to initialise svgo where you actually use it and keep the config to the purest JSON form possible. This will also help us improve this plugin for better caching in the future :)